src/metadataGeneration/customAttribute.ts

```diff
@@ -0,0 +1,67 @@
+import * as ts from 'typescript';
+import { getInitializerValue } from './initializer-value';
+import { Tsoa } from './tsoa';
+
+const getArgumentValue = (argument: ts.Expression) => {
+  switch (argument.kind as ts.SyntaxKind) {
+    case ts.SyntaxKind.StringLiteral: {
+      const argumentValue = (argument as any).text;
+
+      return argumentValue;
+    }
+
+    case ts.SyntaxKind.ObjectLiteralExpression: {
+      const argumentValueProperties = (argument as any).properties;
+
+      const argumentValue = {};
+
+      for (const property of argumentValueProperties) {
+        const name = property.name.text;
```

Comment:
`property.name` may be `undefined`, should be checked.
If it is defined, it can be `Identifier | StringLiteral | NumericLiteral | ComputedPropertyName`.
By using getText(), all of these can be handled correctly, not just `Identifier`